client/dashboard/domains/dataviews/field-domain-name.tsx

```diff
@@ -28,19 +24,27 @@ export const DomainNameField = ( {
 		<Link
 			to={ domainOverviewRoute.fullPath }
 			params={ { siteSlug, domainName: domain.domain } }
-			disabled={ domain.subtype.id === DomainSubtype.DEFAULT_ADDRESS }
+			disabled={ ! domain.subscription_id }
```

Comment:
@hambai this change makes all domains that don't have a subscription_id unclickable. Let me know if that doesn't make sense.